src/components/Cluster/ClusterDetail/EditClusterLabelsModal/AddClusterLabel.tsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Button from 'UI/Button';
+import EditableValueLabel from 'UI/EditableValueLabel';
+
+interface IAddClusterLabelProps {
+  onSave({ label, value }: { label: string; value: string | null }): void;
+}
+
+const AddClusterLabel = ({ onSave }: IAddClusterLabelProps) => {
```

Comment:
There's a more common way to do it
```suggestion
const AddClusterLabel: React.FC<IAddClusterLabelProps> = ({ onSave }) => {
```

Reply:
Btw this is present in multiple places, not only in this component